example/otel-collector/k8s/otel-collector.yaml

```
@@ -105,39 +106,39 @@ spec:
         component: otel-collector
     spec:
       containers:
-      - command:
-          - "/otelcol"
-          - "--config=/conf/otel-collector-config.yaml"
-#           Memory Ballast size should be max 1/3 to 1/2 of memory.
-          - "--mem-ballast-size-mib=683"
-        env:
-        - name: GOGC
-          value: "80"
-        image: otel/opentelemetry-collector:0.3.0
-        name: otel-collector
-        resources:
-          limits:
-            cpu: 1
-            memory: 2Gi
-          requests:
-            cpu: 200m
-            memory: 400Mi
-        ports:
-        - containerPort: 55680 # Default endpoint for otlp receiver.
-        - containerPort: 8889  # Default endpoint for querying metrics.
-        volumeMounts:
-        - name: otel-collector-config-vol
-          mountPath: /conf
-#        - name: otel-collector-secrets
-#          mountPath: /secrets
-        livenessProbe:
-          httpGet:
-            path: /
-            port: 13133 # Health Check extension default port.
-        readinessProbe:
-          httpGet:
-            path: /
-            port: 13133 # Health Check extension default port.
+        - command:
+            - "/otelcol"
+            - "--config=/conf/otel-collector-config.yaml"
+            #           Memory Ballast size should be max 1/3 to 1/2 of memory.
```

Not sure if the space after the `#` is needed anymore.

```suggestion
            # Memory Ballast size should be max 1/3 to 1/2 of memory.
```